lib/Spot/Entity.php

```diff
@@ -207,11 +208,17 @@ public function data($data = null, $modified = true)
                 $v = $this->__get($k, $v);
             }
 
-            foreach (self::$relationFields[$entityName] as $relationField) {
-                $relation = $this->relation($relationField);
+            if ($loadRelations) {
+                foreach (self::$relationFields[$entityName] as $relationField) {
+                    $relation = $this->relation($relationField);
 
-                if ($relation instanceof Entity\Collection || $relation instanceof EntityInterface) {
-                    $data[$relationField] = $relation->toArray();
+                    if ($relation instanceof Entity\Collection) {
+                        $data[$relationField] = $relation->toArray();
+                    }
+
+                    if ($relation instanceof EntityInterface) {
+                        $data[$relationField] = $relation->toArray(false);
```

Comment:
I think I would rather use `$relation->data(null, true, false)` here instead of modifying the signature of `toArray`. The reason is that the signature of `toArray` for collections is different, and I don't want to exclude the possibility of matching that for individual entities later down the road.